specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/configuration.json

```diff
@@ -0,0 +1,264 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerRegistryManagementClient",
+    "description": "Admin Container Registry Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/configuration/{configurationName}": {
```

Comment:
why is the resource name not plural?

Reply:
configuration**s**

Reply:
it is singleton, only 1 item is expected for List

Reply:
I thought it will be configurations/default ?

Reply:
bala is right, this should follow the singleton pattern, meaning .../configurations/default

Reply:
not addressed

Reply:
made the change